app.py

```diff
@@ -214,13 +240,31 @@ def get_repository_readme(project_key: str, repo_slug: str) -> str:
     return readme_content
 
 
+def get_latest_commit(project_key: str, repo_slug: str) -> dict[str, Any]:
+    commit_path = f"projects/{project_key}/repos/{repo_slug}/commits"
+    for commit_batch in get_paginated_resource(path=commit_path, page_size=1):
+        latest_commit = commit_batch[0]
+        return latest_commit
```

Comment:
Please add more users to your bitbucket server and test this function since I doubt it is working as it's intended. 
Is your goal to return just one record from the paginated resource?

Reply:
This is working as expected, we only need the last user that committed